articles/connections/social/fitbit.md

```diff
@@ -11,47 +11,45 @@ index: 14
 New connections with Fitbit will use OAuth 2.0. Please see the following for reference on OAuth 2.0 at Fitbit: https://dev.fitbit.com/docs/oauth2/
```

Comment:
@jeffreylees Could you fix this URL like that? https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#links
Thanks.